build.rs

```diff
@@ -29,13 +27,12 @@ impl bindgen::callbacks::ParseCallbacks for IgnoreMacros {
 }
 
 /**
- * Extract WolfSSL
+ * Copy WolfSSL
  */
-fn extract_wolfssl(dest: &str) -> std::io::Result<()> {
-    Command::new("tar")
-        .arg("-zxvf")
-        .arg(format!("vendor/{WOLFSSL_VERSION}.tar.gz"))
-        .arg("-C")
+fn copy_wolfssl(dest: &str) -> std::io::Result<()> {
+    Command::new("cp")
+        .arg("-rf")
+        .arg("wolfssl")
```

Comment:
Is it a given that `dest` never contains an existing `wolfssl` directory? It's ultimately `env::var("OUT_DIR")` so maybe but I'm honestly not sure?

Previously we'd just have untarred the same thing back over it or if wolfssl changed it would go into a new versioned dir so it was ~harmless, but now I think we might end up making  `$OUT_DIR/wolfssl` on the first build and  `$OUT_DIR/wolfssl/wolfssl` on the second?

I suppose it's too much to hope that wolfssl supports out of tree builds, so maybe this just needs an `rm` first?